src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const paginationItemsCount = Math.ceil(total / perPage);
+  const next = currentPage >= paginationItemsCount;
+  const previous = currentPage === 1;
+
+  const handleNextLinkClick = () => {
+    if (currentPage !== paginationItemsCount) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const handlePreviousLinkClick = () => {
+    if (currentPage !== 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames(['page-item', previous ? 'disabled' : ''])}
+        onClick={handlePreviousLinkClick}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={previous}
+        >
+          «
```

Comment:
Using href="#prev" and href="#next" may cause the page to jump to the top when clicked. It's better to use 'button' elements for previous and next controls or prevent the default action of the anchor tags in the click handlers.